src/js/angular/guides/steps/complex/execute-sparql-query/plugin.js

```diff
@@ -23,17 +23,35 @@ PluginRegistry.add('guide.step', [
                 }
             ];
 
+            const defaultQuery = 'select * where { \n\t?s ?p ?o .\n} limit 100 \n';
+            const queries = {};
+            queries[-1] = 'PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\nselect * where { \n\t?s ?p ?o .\n?o rdf:type ""\n} limit 100 ';
```

Comment:
Why is this needed?

Reply:
This is because of previous flow. When the guide is on the step when first query is executed and user click "previous" button, this query is executed to cleared the result table.